### PR TITLE
feat: added missing regenerate api key method in Environments

### PR DESCRIPTION
**Pull request by mr-palindrome**
See https://docs.novu.co/api-reference/environments/regenerate-api-keys for more details

Part of #74

### CODE REVIEW

File: novu/api/environment.py [hunk 1]
```diff
@@ -67,3 +67,15 @@ def api_keys(self) -> Iterator[EnvironmentApiKeyDto]:
...
+        """Regenerate an environment api key
+
+        Yields:
+            Mapped environment's api key
+        """
+        results = self.handle_request("POST", f"{self._environment_url}/api-keys/regenerate")["data"]
+        if not isinstance(results, list):
+            results = [results]
+        for result in results:
+            yield EnvironmentApiKeyDto.from_camel_case(result)
```
Comment (ryshu) [inline novu/api/environment.py:81]: API docs say that we only retrieve list from them, we should not add this I think

```suggestion
        for result in results:
            yield EnvironmentApiKeyDto.from_camel_case(result)
```

File: tests/api/test_environment.py [hunk 2]
```diff
@@ -129,3 +129,20 @@ def test_list_environment_api_keys(self, mock_request: mock.MagicMock) -> None:
             params=None,
             timeout=5,
         )
+
+    @mock.patch("requests.request")
+    def test_regenerate_api_key(self, mock_request: mock.MagicMock) -> None:
+        mock_request.return_value = MockResponse(200, {"data": self.response_json_api_key})
```
Comment (ryshu) [inline tests/api/test_environment.py:135]:
```suggestion
        mock_request.return_value = MockResponse(200, {"data": [self.response_json_api_key]})
```

**Review by ryshu — CHANGES_REQUESTED** [review]

**Review by ryshu — APPROVED** [review]

File: tests/api/test_environment.py (hunk 2 above)
Comment (mr-palindrome) [reply tests/api/test_environment.py]: Hey @ryshu thanks for reflecting this. I checked this for the response which is coming in the test sample has different format for result, it is not list in that.. I have reflected this thing in discord! Here: https://discord.com/channels/895029566685462578/1160432550301405276
Can you check this from your side!

Comment (mr-palindrome) [reply tests/api/test_environment.py]: Okay doing this works.. thanks for the suggestions!

**Comment by mr-palindrome** [comment]
This PR solves issue #74

**Pull request merged**